### PR TITLE
[AIRFLOW-XXX] [WIP] Enforce order in imports

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below.

### Jira

Will be reported at a later date.

### Description

Under some PR there is a discussion in what order to arrange imports. Unfortunately, without the automation of checking this, it is impossible to ensure the durability of these changes. I am reporting PR. If it receives a positive opinion, I will finish it.

This change does not conflict with the AIP-6, but is its complement.
https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-6+Enforce+the+usage+of+a+code+formatter

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by BasPH** [comment]
Nice! Didn't know it was possible to set this with flake8. +1 for this!

**Comment by Fokko** [comment]
`love-it.py`

File: setup.py [hunk 1]
```diff
@@ -233,6 +233,8 @@ def write_version(filename=os.path.join(*['airflow',
 
 devel = [
     'click==6.7',
+    'flake8-import-order-0.18',
```
Comment (mik-laj) [inline setup.py]:
```suggestion
    'flake8-import-order>=0.18',
```

**Review by feng-tao — APPROVED** [review]

**Comment by feng-tao** [comment]
@mik-laj , anything else we need to update from this pr?

**Comment by mik-laj** [comment]
Yes. This is not finished. Many errors still exist. I need to fix them, but I've been dealing with other project areas that are key to my company. This PR does not have a high priority for me, but I will try to finish by the end of the week.

**Comment by mik-laj** [comment]
This PR is broken. I will create a new PR, because I can not change my status. I do not see the appropriate button that changes the status. 
Screenshot 2019-03-10 at 20 38 10

Draft is an experiment made by Github. It creates more problems than it solves. Travis does not work on draft pull request, which makes testing difficult.  We experimented with this feature in the company and abandoned its use. For now, it does not work properly.

**Pull request closed without merging**